client/dist/src/components/app.jsx

```diff
@@ -3,25 +3,37 @@ import Bars from './bars.jsx';
 import FiftyTwoWeekInfo from './fiftyTwoWeekInfo.jsx';
 import CurrentPrice from './currentPrice.jsx';
 import axios from 'axios';
+import styles from '../css/app.css';
 
 
+const marketIsOpen = () => {
+  var date = new Date();
+  var currentHour = date.getHours();
+  console.log(currentHour);
+  if (currentHour > 6 && currentHour < 15) {
+    return true;
+  } else {
```

Comment:
Stylistically, it may be a benefit to just return false rather than make an if statement. Up to you though, more of a matter of preference.